libraries/AP_Logger/AP_Logger_File.cpp

```diff
@@ -837,6 +840,9 @@ void AP_Logger_File::start_new_log(void)
     _need_rtc_update = !AP::rtc().get_utc_usec(utc_usec);
 #endif
 
+    // create the log directory if need be
```

Comment:
if you think you need a comment here, I would have put it in the .h file.  Again, not a blocker ..